src/component/ShortBreakTimer/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const ShortBreakTimer = () => {
+    return (
+        <div>
+            
+        </div>
+    )
+}
+
+export default ShortBreakTimer
```

Comment:
For now you can leave this the way it is. But in the future, you should not have separate components for the short break timer, the long break timer, and the Pomodoro timer. They should all be one timer component and just change the number displayed.

Reply:
despair